app/views/layouts/application.html.erb

```diff
@@ -12,6 +12,10 @@
     <meta name="author" content="Public Lab contributors" />
     <link href="https://<%= request.host %>/feed.rss" rel="alternate" type="application/rss+xml" title="Public Lab research" />
 
+   
+    <script src="https://maps.googleapis.com/maps/api/js?libraries=places&language=en&key=AIzaSyAOLUQngEmJv0_zcG1xkGq-CXIPpLQY8iQ"></script>
```

Comment:
Hi @jywarren ! Instead of adding the **if condition** in `_footer.html.erb` , can i add that if condition here ? 
When used in footer , the Google API gave - _google not found error_ and when adding an explicit script tag to the page - it gave - 'google api used two times , may give unexpected error' . 

So i finally removed the google API script everywhere AND kept it here . What do you think ?

Reply:
Sure, that sounds good. Thanks!